packages/plugin/src/PluginTypes.ts

```diff
@@ -1,4 +1,9 @@
 import type { BaseThemeType } from '@deephaven/components';
+import { type JsWidget } from '@deephaven/jsapi-types';
```

Comment:
I thought it had to be `import type { JsWidget } from '@deephaven/jsapi-types';`. But I guess this works too? Neat. But also inconsistent with the line immediately before it.

Reply:
Yep at some point this became valid syntax (looks like TS 4.5 added the inline types). It lets you combine type and non-type named imports on the same line. In some cases we have

```ts
import { a } from 'abc';
import type { b } from 'abc';
```

which can now be

```ts
import { a, type b } from 'abc';
```

There's a few related rules. Should I create 2 tickets for them?
1 for [import type specifier](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/consistent-type-specifier-style.md) and 1 for [consistent imports and exports](https://typescript-eslint.io/blog/consistent-type-imports-and-exports-why-and-how/#enforcing-with-typescript-eslint)? The 2nd will likely require more touched files, but I think it's auto-fixable

Reply:
Sure